build-system/babel-plugins/babel-plugin-transform-amp-asserts/test/fixtures/transform-assertions/transform-devAssert-false/output.js

```diff
@@ -13,6 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-(function () {
-  throw new Error('static assertion failure');
-})();
+devAssert(false);
```

Comment:
Oo, this is a bad change. We still need to transform `devAssert` only for the statically-known-falsey case into a throw.

Reply:
@jridgewell I'm not too familiar with babel tranforms, so I'll let you determine what needs to change in order to accomplish this. Any chance you can add a commit for this?

Also, if it's feasible, I think it makes sense to move all the babel changes to a separate PR authored by you. I can follow up with a PR that removes `AmpPass.java`.

Reply:
This was done in #26357. Will remove all babel changes in this PR and rebase.